apisix/plugins/error-log-logger.lua

```diff
@@ -180,10 +180,15 @@ local function send_to_skywalking(log_message)
     httpc:set_timeout(config.timeout * 1000)
 
     local entries = {}
+    local serviceInstanceName = config.skywalking.service_instance_name
```

Comment:
We should call it in the `init` method, instead of in the per-report way

Reply:
I change my mind. It is a route-level configuration and may be changed on the fly.